config_example.yml

```diff
@@ -4,14 +4,14 @@ github_organization: ""
 # Cloud connection infos
 cloud_name: 'openstack'
 cloud_config:
-    auth_url: ""
-    region_name: ""
-    project_name: ""
-
-    username: ""
-    password: ""
-    token: ""
-    network_name: ""
+  auth_url: ""
+  region_name: ""
+  project_name: ""
+
+  username: ""
+  password: ""
+  token: ""
+  network_name: ""
```

Comment:
This is Openstack cloud config, we should delete it and link to the cloud folder :thinking: